hack/test-cmd.sh

```diff
@@ -443,14 +443,15 @@ osc delete all -l app=guestbook
 echo "edit: ok"
 
 osc delete all --all
-osc new-app https://github.com/openshift/ruby-hello-world -l app=ruby
```

Comment:
> ok. as long as the new-app command itself is exercised

It is: https://github.com/openshift/origin/pull/2611/files#diff-e2c8544666bb21f92286b190d5c8cb9aR407